jaxrs/src/main/java/org/killbill/billing/jaxrs/json/CatalogJson.java

```diff
@@ -375,22 +426,204 @@ public int hashCode() {
         }
     }
 
+    public static class TieredBlockJson {
```

Comment:
@pierre 
I've cloned the **killbill-java-parser** project and tried with this command:
```
java -cp target/killbill-java-parser-0.0.1-SNAPSHOT.jar com.ning.killbill.KillbillParser -i file:///$KILLBILL_REPO/jaxrs/src/main/java/ -o /tmp/output -t RUBY_CLIENT_API -p org.killbill.billing.jaxrs.json
```

Replacing the $KILLBILL_REPO variable with my location. But I got this error:
```
Error: Could not find or load main class com.ning.killbill.KillbillParser
```

I've tried to compile the project using `mvn clean install` but there is an error:
```
[ERROR] Failed to execute goal org.antlr:antlr4-maven-plugin:4.0:antlr4 (default) on project killbill-java-parser: Execution default of goal org.antlr:antlr4-maven-plugin:4.0:antlr4 failed: Index: 0, Size: 0 -> [Help 1]
```

Do I have to clone / import anything else?

Thanks!!